### ISSUE TITLE
how to let the connection to use charset utf8 so I can deal with non ASCII characters?

### ISSUE THREAD

**Issue by Michael-Jing** [JuliaDB/MySQL.jl#46]
how to let the connection to use charset utf8 so I can deal with non ASCII characters? please help

**Comment by nkottary** [comment]
MySQL supports UTF-8 by default.  If this is not the case for you then please edit the `/etc/mysql/my.cnf` as per the instructions [here](http://stackoverflow.com/questions/3513773/change-mysql-default-character-set-to-utf-8-in-my-cnf).  You will have to restart the server after this.

**Comment by Michael-Jing** [comment]
I tried the suggestions and still have issue to display Chinese characters correctly. Look at the following python code, in which the "charset=utf8' part solves the problem. Looking into the code of hany.jl, I found that the mysql_connect function has parameter opts = Dict(), can we put something there maybe can solve the problem?
engine = sqlalchemy.create_engine("mysql://user:pass@localhost/ganji?charset=utf8")
